src/aura/STG_CMP_Tabs/STG_CMP_TabsHelper.js

```diff
@@ -383,29 +381,6 @@
         }else{
             return false;
         }
-    },
-
-    getAdminAccNameFormatOptions : function(component) {
```

Comment:
I thought we recently moved these methods here to avoid calling them twice. (Work that Xinnan originally did), or has this branch been open long enough that this is in fact making it match?

Reply:
Thanks for catching this. I went to the master branch and copied the source code for STG_CMP_TabsHelper.js from there.